setup/productionize/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: single
+title:  "Productionize Spinnaker"
+sidebar:
+  nav: setup
+---
+
+Spinnaker is a large system, made of many microservices, each intended to be
+scaled, restarted, and configured independently. This provides operators a
+great degree of flexibility, and allows Spinnaker to handle massive scale (1K+
+deployments/day, 10K+ managed machines). However, there is no one-size-fits-all
+approach for configuring Spinnaker; your organization's usage patterns will
+need to inform how to productionize your Spinnaker deployment.
```

Comment:
I'm ok with "productionize" in the title and headings, but maybe in the first mention in text say "...infor how to prepare your Spinnaker deployment to be used in production."

Reply:
SGTM